scripts/test/run.py

```diff
@@ -6,7 +6,7 @@
 
 
 def mypy_src():
-    cmd = ["mypy", "pandas-stubs", "tests", "--no-incremental"]
+    cmd = ["mypy", "pandas-stubs", "tests"]
```

Comment:
It doesn't seem to fail anymore :) Running `poe mypy` twice takes for me 44.882s and then 0.710s.